src/styles.scss

```diff
@@ -88,3 +88,8 @@
 .fb_customer_chat_bubble_pop_in {
   bottom: 100px !important;
 }
+
+.positionSticky {
+  position: sticky;
+  top: 40px;
```

Comment:
<img width="1715" alt="image" src="https://github.com/urfit-tech/lodestar-app/assets/64723534/827edeba-1bca-4017-9e49-8361a902d155">

The plan card is still overlapped when it's fixed. 
Is it necessary to hard-code the number of pixels here, instead of automatically detecting the spacing to the top?
If the number of pixels are hard-coded, this issue will keep occurring, since we can't promise this value will fit everywhere.

Reply:
我根據各個產品方案上面的間距分別來設置，要不然沒其他辦法了